.github/workflows/build_iso.yml

```diff
@@ -35,24 +30,28 @@ jobs:
             
     steps:
       - name: Build ISOs
-        uses: ublue-os/isogenerator@1.0.9
+        uses: jasonn3/build-container-installer@v1.1.1
         id: build
         with:
-          ARCH: ${{ contains(matrix.image_tag, 'amd64') && 'x86_64' || 'arm64' }}
-          IMAGE_NAME: ${{ matrix.image_name }}
-          IMAGE_REPO: ghcr.io/ublue-os
+          arch: ${{ contains(matrix.image_tag, 'amd64') && 'x86_64' || 'arm64' }}
```

Comment:
building ARM images isn't supported in upstream fedora or in the upstream image builder.

Reply:
That was just me being hopeful and making it as easy as possible to implement once support is available.  
The `contains` condition is doing what it's supposed to, and only the amd64 (x86_64) images are building.
![image](https://github.com/ublue-os/cosmic/assets/46304672/1bca07b8-4ee7-4a64-baf6-0e4d3920b5cb)